src/app/cli/src/database.rs

```diff
@@ -229,3 +232,24 @@ fn init_database_password_provider(b: &mut CatalogBuilder, raw_db_config: &Datab
 }
 
 ////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+static SQLITE_MIGRATOR: Migrator = sqlx::migrate!("../../../migrations/sqlite");
```

Comment:
Is this embedding migration files into the executable?

Reply:
Yes